aas_core_meta/v3.py

```diff
@@ -24,6 +24,10 @@
 
 * :constraintref:`AASd-012`
 
+:constraintref:`AASd-116`: In this constraint the string ``globalAssetId`` is checked case-sensitively,
```

Comment:
@s-heppner It is not clear from the text what the book says and where we diverge.

Suggestion: "In the book, :constraintref:`AASd-116` imposes a case-*insensitive* equality against ``globalAssetId``. This is culturally-dependent, and depends on the system settings. For example, the case-folding for the letter "i" and "I" is different from English.

We implement the constraint as case-*sensitive* instead to allow for interoperability across different culture settings.

Reply:
@s-heppner this constraint could also be approximated as `[gG][lL][oO]...`. Is that not an option? Obviously, this works only for some languages.

Reply:
Both ideas are really good, imo. I'll deliberate with the team which we would want to choose.

Reply:
👍

Reply:
We decided, choosing your first suggested approach.

Reply:
Fixed the missing part of the sentence:

 "In the book, :constraintref:`AASd-116` imposes a case-_insensitive_ equality against `globalAssetId`. This is culturally-dependent, and depends on the system settings. For example, the case-folding for the letters "i" and "I" is different in Turkish from English.

We implement the constraint as case-_sensitive_ instead to allow for interoperability across different culture settings."